filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
```

Comment:
I had edited the previous comment right after I posted it. I missed the point you were trying to make. You are correct here.